rudderjsonadapter/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
```

Comment:
where is the entry for v1.0.1?

Reply:
I have deleted it. We haven't released any. I was just testing